c7n/resources/ec2.py

```diff
@@ -652,6 +652,83 @@ def process_volume_set(self, resource):
                     response['SnapshotId']],
                 Tags=tags)
 
+@actions.register('rename-tag')
+class RenameTag(BaseAction):
+
+    schema = type_schema(
+        'rename-tag',
+        map={
```

Comment:
Perhaps I misinterpreted something you wrote in an earlier comment regarding this... I can revert this back.